src/frontends/tensorflow_lite/include/openvino/frontend/tensorflow_lite/quantization_info.hpp

```diff
@@ -15,13 +15,13 @@ namespace ov {
 namespace frontend {
 namespace tensorflow_lite {
 
-class TENSORFLOW_LITE_API QuantizationInfo : public ov::RuntimeAttribute {
+class TENSORFLOW_LITE_FRONTEND_C_API QuantizationInfo : public ov::RuntimeAttribute {
 public:
     OPENVINO_RTTI("QuantizationInfo", "0", RuntimeAttribute);
     QuantizationInfo() = default;
-    explicit QuantizationInfo(const std::vector<float>& scale,
-                              const std::vector<int64_t>& zero_point,
-                              const int64_t& axis)
+    QuantizationInfo(const std::vector<float>& scale,
```

Comment:
why changed?

Reply:
because `explicit` is used with ctors having a single argument